Resources/public/javascript/cropper.js

```diff
@@ -0,0 +1,24 @@
+let cropTrigger;
+
+class cropSave {
```

Comment:
Try to use the same class name as the selector at the bottom, eg: .cropper -> Cropper.